tests/bdd/word_document_generator/generate_document.feature

```diff
@@ -0,0 +1,41 @@
+@word_document_generator
+Feature: Generate Word document from template
+
+    Scenario: Generate Word document from template
+        Given I have a template file 'template1.docx'
+        And I read the template context from 'context1.json'
+        When I generate a Word document named 'output1.docx'
+        Then the Word document should be generated
+        # We should also check the content of the generated document but it's not easy to do
```

Comment:
Is the rendering process deterministic?

If so, we could run this once, manually verify the output and then store it alongside the input and output folders. Then we could validate the test output by loading the expected output and comparing byte by byte.

Reply:
I did wonder that, but I didn't know if there was any metadata stored in the file that might throw the comparison off. I can try it out.

Reply:
It seems it's not deterministic, presumably due to metadata